bigframes/dataframe.py

```diff
@@ -752,7 +752,7 @@ def _apply_binop(
         )
 
     def _apply_scalar_binop(
-        self, other: float | int, op: ops.BinaryOp, reverse: bool = False
+        self, other: float | int | bool | str, op: ops.BinaryOp, reverse: bool = False
```

Comment:
SHould we use `bigframes.dtypes.LOCAL_SCALAR_TYPES` here too?

Reply:
Updated, added a new constant for annotation.